reconcile/terraform_resources.py

```diff
@@ -606,6 +606,7 @@ def populate_desired_state(
                 resource_type=oc_resource.kind,
                 name=spec.output_resource_name,
                 value=oc_resource,
+                privileged=spec.namespace.get("clusterAdmin", False),
```

Comment:
when `clusterAdmin` is not defined in a namespace file, the graphql query will return null for it and `get("clusterAdmin", False)` will result in None. 

```suggestion
                privileged=spec.namespace.get("clusterAdmin") or False,
```

Reply:
You are right. Even though the result is the same is better set this way. 
Fixed.

Reply:
absolutely. but explicit types are always nice :)
thanks for fixing it!